src/core/auth/jwt.strategy.ts

```
@@ -8,6 +8,8 @@ import { User } from '@prisma/client';
 
 import { AuthUnauthorizedException } from './exception/auth.exception';
 
+import { User } from '@prisma/client';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)